app/src/main/AndroidManifest.xml

```diff
@@ -16,17 +17,25 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/Theme.Morelocale">
-        <activity android:name=".ui.MainActivity">
+        <activity
+            android:name=".ui.MainActivity"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <receiver android:name=".receiver.BootCompletedReceiver">
+        <receiver
+            android:name=".receiver.BootCompletedReceiver"
+            android:enabled="true"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.BOOT_COMPLETED" />
+                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+                <action android:name="android.intent.action.REBOOT"/>
```

Comment:
`BootCompletedReceiver` filtered intent by `BOOT_COMPLETED`. Should the filter be removed?

https://github.com/c-lis/morelocale/blob/8ecbc19a69bf501f540fdfcade6463018f1b2420/app/src/main/java/jp/co/c_lis/ccl/morelocale/receiver/BootCompletedReceiver.kt#L15-L23

Reply:
Yes, you're right!

Reply:
But direct-boot may be interesting? https://developer.android.com/training/articles/direct-boot